Assets/MRTK/SDK/Experimental/TouchSlider/TouchSlider.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.UI;
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
+using UnityEngine;
+using TMPro;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    /// <summary>
+    /// This control enables touching a slider with one finger to set its value
+    /// </summary>
+    public class TouchSlider : MonoBehaviour, IMixedRealityTouchHandler
+    {
+        [Experimental]
+        [SerializeField]
+        [Tooltip("The pinch slider that will be receiving touch input")]
+        private PinchSlider Slider = null;
```

Comment:
Following [naming conventions](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#naming-conventions) please make private fields camelCase

Reply:
Pushed updates to this script, including coding guidelines updates